File: src/main/java/org/jboss/qa/junitdiff/model/SuperGroup.java

package org.jboss.qa.junitdiff.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Super-group - to cluster multiple groups.
 * Used to bind multiple groups (collumns) visually.
 *
 * @author Ondrej Zizka
 */
public class SuperGroup {

	private final String name;
	private String color = null;

	private List<GroupPrecept> groups = new ArrayList();



	public SuperGroup(String name) {
		this.name = name;
	}



	// <editor-fold defaultstate="collapsed" desc="get/set">
	public String getColor() {
		return color;
	}

	public void setColor(String color) {
		this.color = color;
	}

	public String getName() {
		return name;
	}

	// </editor-fold>


}// class

File: src/main/java/ch/zizka/junitdiff/model/SuperGroup.kt
package ch.zizka.junitdiff.model

/**
 * Super-group - to cluster multiple groups.
 * Used to bind multiple groups (collumns) visually.
 *
 * @author Ondrej Zizka
 */
@Deprecated("Probably not used anywhere.")
class SuperGroup( // </editor-fold>
    val name: String
) {

    // <editor-fold defaultstate="collapsed" desc="get/set">
    var color: String? = null
    private val groups: List<GroupPrecept?> = ArrayList()
}
